wallet/wallet-rpc-lib/src/cmdline.rs

```diff
@@ -96,6 +96,10 @@ pub struct WalletRpcDaemonChainArgs {
     #[arg(long, value_name("ACC_NUMBER"), requires("wallet_file"))]
     start_staking_for_account: Vec<U31>,
 
+    /// use the wallet without a connection to a node
+    #[arg(long, conflicts_with_all(["node_rpc_address", "node_rpc_cookie_file", "node_rpc_username", "node_rpc_password"]))]
```

Comment:
Also `start_staking_for_account`?

Reply:
Also, shouldn't this be a flag, instead of only a boolean?

Reply:
added start_staking_for_account